build-tools/debian-metadata/oss-xamarin.android-targets.install

```diff
@@ -0,0 +1,11 @@
+bin/Release/bin/*					/usr/bin
```

Comment:
Do you want me to try and make those changes, or are they unrelated from this PR?